com.ibm.wala.cast.js.rhino/src/test/java/com/ibm/wala/cast/js/rhino/callgraph/fieldbased/test/FieldBasedCGGamesTest.java

```diff
@@ -13,79 +13,61 @@ public class FieldBasedCGGamesTest extends AbstractFieldBasedTest {
 
   @Test
   public void testBunnyHunt() throws IOException, WalaException, Error, CancelException {
-    runTestExceptOnTravis(
-        new URL("http://www.themaninblue.com/experiment/BunnyHunt/"),
-        new Object[][] {},
-        BuilderType.OPTIMISTIC);
+    URL url = new URL("http://www.themaninblue.com/experiment/BunnyHunt/");
```

Comment:
I would prefer to have all these tests that load the input from a URL to _not_ run on CI; it's just a recipe for flakiness.  Can we somehow keep them disabled on CI?

Reply:
Agreed. Implemented in cac148ea66f9876b1021a9784f078a6b9314bb41.